node/src/components/small_network.rs

```diff
@@ -90,6 +90,7 @@ use crate::{
     components::{
         network::ENABLE_LIBP2P_NET_ENV_VAR, networking_metrics::NetworkingMetrics, Component,
     },
+    e,
```

Comment:
Maybe make it a local macro if it's only going to be used in small_network, as per @Fraser999 's suggestion, which I agree with.

Reply:
This usually gets us in trouble because doctests and tests need it available externally, IIRC.

Reply:
So it can't be entirely local (no `#[macro_export]`) since I do not want the `impl` in the root `small_net` module.